lib/mri/mkmf.rb

```diff
@@ -2191,6 +2191,15 @@ def depend_rules(depend)
   # +VPATH+ and added to the list of +INCFLAGS+.
   #
   def create_makefile(target, srcprefix = nil)
+    
+    # Modified for TruffleRuby
+    unless system("#{RbConfig::CLANG} --version > /dev/null")
+      raise 'clang does not appear to be available - you may need to install LLVM - see doc/user/installing-llvm.md'
+    end
+    unless system("#{RbConfig::OPT} --version > /dev/null")
+      raise 'opt does not appear to be available - you may need to install LLVM - see doc/user/installing-llvm.md'
+    end
+    
```

Comment:
Could you do it with the patching system to keep the file pristine.

Reply:
I'm not sure what patching system you mean - this file already has seven other inline modifications and always has done.

Reply:
never-mind then, I thought we have the `mri/*` files without modifications.